src/components/ActionModal.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable max-len */
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {
+  StyleSheet, Text, View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import Modal from 'react-native-modal';
+import NewHathorButton from './NewHathorButton';
+
+const FeedbackModal = (props) => (
+  <Modal
```

Comment:
If the idea is to create another component similar to the `FeedbackModal`, I prefer if we reuse the previous one to create the new, so we don't need to change more than one place if we decide to improve the UI.

For instance, in this case, we would need to add a `prop` for the `FeedbackModal` that is optional and receives an element to add below the text. Is that correct?

Reply:
It is not so simple. They have different semantics and different structure. They are similar, but only it.

Reply:
Pedro, as we talked I refactored the modals in following commits:
- https://github.com/HathorNetwork/hathor-wallet-mobile/pull/227/commits/7c7d855c587770b9aced78e8d963700d75799746
- https://github.com/HathorNetwork/hathor-wallet-mobile/pull/227/commits/6a448d5d4ed24ffe86bfd5302886b895e5a5930b